app/src/main/java/com/distillery/interview/ui/current_weather/CurrentWeatherViewModel.kt

```diff
@@ -0,0 +1,76 @@
+package com.distillery.interview.ui.current_weather
+
+import android.os.Bundle
+import androidx.lifecycle.*
+import androidx.savedstate.SavedStateRegistryOwner
+import com.distillery.interview.data.api.WeatherAPI
+import com.distillery.interview.data.models.WeatherResponse
+import com.distillery.interview.util.Event
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+
+class CurrentWeatherViewModel(
+    private val weatherAPI: WeatherAPI
+) : ViewModel() {
+
+    private val _uiState = MutableLiveData<CurrentWeatherUiModel>()
+    val uiState: LiveData<CurrentWeatherUiModel>
```

Comment:
The simplest way is just assing the value, e.g.: `val uiState: LiveData<CurrentWeatherUiModel> = _uiState`.